tools/ea/Ea/Genesis.hs

```diff
@@ -147,6 +159,12 @@ developmentN = development0
     & txChainId .~ N
     & coinbase .~ (Just devNGrants)
 
+developmentKAD :: Genesis
+developmentKAD = development0
```

Comment:
more forgivable here to use a default but maybe just build it from scratch too for less confusion than using `development0`

Reply:
Addressed